### DISCUSSION TITLE
Suddenly not able to sync anymore

### DISCUSSION THREAD

**Discussion by DeeJay58** [Chia-Network/chia-blockchain#2994] [Q&A]
Hi!
Maybe anybody's got an idea. After weeks of working it is "not synced" anymore. I did no changes. The only thing happened at the same time is, that my provider changed my iP-Adress. 

Restarting, reinstalling, deleting and reinstalling, reboot rooter does not solve the problem.

What should I try next? Port is open.

Thank you

**Comment by dorkmo** [comment]
make sure to update to newest software

**Reply by DeeJay58** [reply]
Thank you. I've got the newest one since 2 or three days. 
I was able to solve it only by deleting the SQL files. Now it is redownloading the blockchain and I hope then it will sync again.
But this is not really a solution for the future.
